playwright.config.ts

```diff
@@ -0,0 +1,31 @@
+import { defineConfig, devices } from '@playwright/test';
+
+// See https://playwright.dev/docs/test-configuration.
+export default defineConfig({
+  testDir: './e2e',
+  // 1 minute, default is 30 seconds
+  timeout: 60 * 1000,
+  // Run tests in files in parallel
+  fullyParallel: true,
+  // Fail the build on CI if you accidentally left test.only in the source code
+  forbidOnly: !!process.env.CI,
+  // Retry on CI only
+  retries: process.env.CI ? 2 : 0,
+  // Opt out of parallel tests on CI
+  workers: process.env.CI ? 1 : undefined,
```

Comment:
How does this aligns with `fullyParallel: true`?

Reply:
as I understand:
`fullyParallel: true` enables parallel tests _inside_ a single file
`workers: process.env.CI ? 1 : undefined` ensures only one file runs at a time in CI.
so it kinda limits file-level concurrency in CI but keeps parallelism within file